### PR TITLE
Feature/simple mixture predictor

### PR DESCRIPTION
**Pull request by gregor-robinson**
# Citrine Python PR

## Description 
Please briefly explain the goal of the changes/this PR.

### PR Type:
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

### Adherence to team decisions
- [x] I have added tests for 100% coverage
- [x] I have written Numpy-style docstrings for every method and class.
- [x] I have communicated the downstream consequences of the PR to others.
- [ ] I have bumped the version in setup.py

### CODE REVIEW

File: docs/source/workflows/predictors.rst [hunk 1]
```diff
@@ -214,6 +214,34 @@ The following example illustrates how an :class:`~citrine.informatics.predictors
         }
     )
 
+Simple mixture predictor
+---------------------------------------
+
+Simple mixtures may contain ingredients that are blends of other simple mixtures.
+Along the lines of the example above, hypertonic saline can be mixed with water to form isotonic saline.
+Often, the properties of a hierarchical mixture are strongly associated with its atomic ingredients.
```
Comment (gregor-robinson) [inline docs/source/workflows/predictors.rst:222]: I notice now that we should probably avoid the word "atomic" lest it be confused with elemental atoms. I'll change to something like "indivisible" -- @latture does that sit well with you?

Comment (latture) [reply docs/source/workflows/predictors.rst:222]: I think "leaf ingredients" would work here.

**Comment by latture** [comment]
@gregor-robinson There are a few merge conflicts after my last merge from `master` into `feature/formulations`. I've already resolved similar issues in #324 and #332, so I can fix the conflicts and push up the changes. Sorry for shifting the base on you.

**Comment by latture** [comment]
@maxhutch @genfx999 Similar to #332, I've reviewed this PR and think it looks good. However, I've pushed enough code to this branch that it would be good to get a second set of eyes. Can one of you review when you get a chance? Note the version bump to 0.25.0. I'm assuming we'll merge this after #332.

**Pull request merged**